README.md

```diff
@@ -8,34 +8,19 @@
 [![Coverage Status](https://codecov.io/gh/aws/aws-sdk-js/branch/master/graph/badge.svg)](https://codecov.io/gh/aws/aws-sdk-js)
 [![Known Vulnerabilities](https://snyk.io/test/github/aws/aws-sdk-js/badge.svg)](https://snyk.io/test/github/aws/aws-sdk-js)
 
-**Note**: The [version 3.x](https://github.com/aws/aws-sdk-js-v3) of the AWS SDK for JavaScript is generally available. For more information see the [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/) or [API Reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html).
+## VERSION 3.x NOW AVAILABLE
```

Comment:
[ALL CAPS](https://en.wikipedia.org/wiki/All_caps) are sometimes referred to as "screaming" or "shouting". Avoid using them.
```suggestion
## Version 3.x now available
```

Reply:
Changed